package.json

```diff
@@ -5,11 +5,14 @@
   "author": "Diederik van den Burger <diederikvandenburger@tab.capital>",
   "repository": "https://github.com/DiederikvandenB/apollo-link-sentry.git",
   "homepage": "https://github.com/DiederikvandenB/apollo-link-sentry",
-  "main": "lib/index.js",
-  "typings": "lib/index.d.ts",
+  "main": "lib-cjs/index.js",
+  "module": "lib-esm/index.js",
+  "browser": "lib-esm/index.js",
+  "typings": "lib-esm/index.d.ts",
   "files": [
     "lib/**/*"
```

Comment:
Also no longer accurate, is it even needed?

Reply:
Missed that, thanks

> is it even needed

I think yes. Since it makes a package smaller